### ISSUE TITLE
pytz isn't installed

### ISSUE THREAD

**Issue by nokster1001** [echocat/puppet-graphite#233]
Hi,
graphite failed after installation with the following:
```bash
traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/opt/graphite/webapp/graphite/render/views.py", line 56, in renderView
    (graphOptions, requestOptions) = parseOptions(request)
  File "/opt/graphite/webapp/graphite/render/views.py", line 312, in parseOptions
    untilTime = parseATTime(queryParams['until'], tzinfo, now)
  File "/opt/graphite/webapp/graphite/render/attime.py", line 48, in parseATTime
    return tzinfo.normalize(parseTimeReference(ref).astimezone(tzinfo) + parseTimeOffset(offset))
AttributeError: 'LocalTimezone' object has no attribute 'normalize'
```
installing pytz solved it:
```bash 
pip install pytz 
```
see: https://github.com/graphite-project/graphite-web/issues/1019
I think pytz should be added to the module, no?

**Comment by dwerder** [comment]
Which OS do you use? And how do you use this module? Do you use it plain or with own python installation modules?

**Comment by nokster1001** [comment]
CentOS 7
and passing the relevant manifest part here:
```puppet
 class {'::graphite':
    gr_graphite_ver    => '0.9.15',
    gr_carbon_ver      => '0.9.15',
    gr_whisper_ver     => '0.9.15',
    gr_web_server      => 'wsgionly',
	...
    gr_django_pkg      => 'django',
    gr_django_ver      => '1.6',
    gr_django_provider => 'pip',
    gr_storage_schemas => [
	....
  }
```

**Comment by dwerder** [comment]
Fixed by bdd7e2c

**Comment by nokster1001** [comment]
thanks ! :)
